src/App.js

```diff
@@ -1,18 +1,15 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
-import './App.css';
 
 import HomePage from './pages/homepage/homepage.component';
 import DetailsPage from './pages/details/details.component';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
`App` is not used as a class in css. No need for this `<div>`

Reply:
Removed